Update manpages (#50)
aviator-co/gitbook-docs: aviator-cli/manpages/av-stack-reparent.1.md

# av-stack-reparent

## NAME

> av-stack-reparent - Reparent a branch to a new parent

Reviewer: ```suggestion
av-stack-reparent - Change the parent of the current branch
```

## SYNOPSIS

```synopsis
av stack reparent [--parent=<parent>]
```

## DESCRIPTION

This rebases the current branch onto the new parent and runs the restack

> operations on the children.

Reviewer: ```suggestion
operations on the children. It does not push the changes to remote.
```
